continuous-integration/linux/platform-dependent/ubuntu-23.04/setup.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+export DEBIAN_FRONTEND=noninteractive
+SCRIPT_DIR="$(realpath $(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd) )"
+
+function apt-yes {
+    apt-get --assume-yes "$@"
+}
+
+apt-yes update || exit $?
+apt-yes dist-upgrade || exit $?
+
+apt-yes install \
+    clinfo \
+    g++ \
+    jq \
+    python3-dev \
+    python3-venv \
+    python3-pip \
+    wget \
+    || exit $?
+
+update-alternatives --install /usr/bin/python python /usr/bin/python3 0 || exit $?
+update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 0 || exit $?
+
+source $(realpath $SCRIPT_DIR/../common.sh) || exit $?
+ubuntu_install_opencl_cpu_runtime || exit $?
+
+function install_www_deb {
+    TMP_DIR=$(mktemp --tmpdir --directory zivid-python-install-www-deb-XXXX) || exit $?
+    pushd $TMP_DIR || exit $?
+    wget -nv "$@" || exit $?
```

Comment:
```suggestion
    wget --no-verbose "$@" || exit $?
```
I had to look up this argument, actually. I think it is slightly easier to understand

Reply:
This is an existing issue (i just moved this code around), so I'll leave it to a follow-up.